DiverseSelector/methods/dissimilarity.py

```diff
@@ -55,62 +61,71 @@ def __init__(self, func_distance=None):
         Parameters
         ----------
         func_distance: callable
```

Comment:
Can you help address this problem throughout this PR because we will need one extra white space before the colon? For example,

```rst
var1 : array_like
```